docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.ts

```diff
@@ -36,12 +36,13 @@ export class CssUxIconsComponent {
 
     size: string = '24px';
 
+    hpeIconsLinkHtml = require('!!raw-loader!./hpe-icons-link.txt');
```

Comment:
I followed this method of loading a code example that I found in other docs, let me know if this is not common practice.

Reply:
See comment above above how we handle snippets